src/test/resources/features/RFMFinalReviewCYAPage.feature

```diff
@@ -0,0 +1,142 @@
+@tests
+Feature: RFM Final Review check your answers page
+  As a replacing filing member
+  I want to enter my subscription details and access Final Review CYA page to check my answers and change
+  @zap_accessibility
+  Scenario: 1 - Validating RFM Check your answers page
+    Given Organisation User logs in with rfm URL to Pillar2
+    And I access RFM corporate position page
+    When I select corp position as UPE
+    And I click on Continue button
+    Then I should be on RFM Contact Guidance page
+    And I click on Continue button
+    Then I should navigate to RFM Contact Detail Page
+    When I provide RFM contact name as RFM test contact
+    And I click on Continue button
+    Then I should navigate to RFM Contact Email Page
+    When I provide RFM contact email as rfm@email.com
+    And I click on Continue button
+    Then I should navigate to RFM Contact Number Page
+    When I select option No and continue to next
+    Then I should navigate to RFM Second Contact Question Page
+    And I click on Continue button
+    Then I should navigate to RFM Second Contact Question Page
+    And I select option No and continue to next
+    Then I should be on RFM Second Contact Address Page
+    When I enter Address Line 1 as RFM Address Line 1
+    And I enter City as RFM City
+    And I enter Postal Code as EH5 5WY
+    And I enter Country as United Kingdom
+    And I click on Country selected
+    When I click on Continue button
+    Then I should navigate to RFM Final Review Page
+    And The Heading should be Review and submit
```

Comment:
need to remove content verification steps and make sure that these are covered in unit tests as per HMRC test leads suggestion